dvc/command/add.py

```diff
@@ -53,6 +53,7 @@ def add_parser(subparsers, parent_parser):
         "-f",
         "--file",
         help="Specify name of the DVC-file this command will generate.",
+        metavar="<filename>",
```

Comment:
I'm not sure how this is better than `-f FILE`. `<>` look out of place here, and I don't really see the value, sorry.

Reply:
But ok, if you want to, sure, lets do it this way. Please remove "Draft" flag.

Reply:
I think, the value can be better seen here - https://dvc.org/doc/command-reference/run#run

check with what `dvc run` outputs.

Reply:
Ruslan, 2 benefits (besides matching the docs which we already changed 😅):

- The existing param names aren't descriptive. FILE isn't the best example of this but e.g. `-o OUTS` why repeat the option name when the param name can hint to what type of argument is accepted.
- We didn't like how upper case looked in the docs. Looks like we're yelling haha. We checked Git and noticed they use `<>` e.g. https://git-scm.com/docs/git-commit